### ISSUE TITLE
Actual minimum version?

### ISSUE THREAD

**Issue by DiscoPanda84** [nikola-kocic/webext-open-link-in-browser-as#1]
I recently updated from Firefox 45.9.0 to Firefox 52.2.0, and I found this extension while looking for e10s-compatible versions of the e10s-incompatible extensions I have installed. (I'm using "Add-on Compatability Reporter" to show which ones are/aren't e10s-compatible.)

The page for this extension said that the minimum version is 53.0, but it does seem to install and work after editing manifest.json to change strict_min_version from "53.0" to "52.2.0".

The downsides to this approach are that the browser shows it as "unverified" (not a real biggie, I have xpinstall.signatures.required set to "false" and the "Nightly Tester Tools" extension installed anyways), but more importantly, I suspect that any updates that may be made are likely to need the same manual installation, and in the meantime it probably won't let me know that there's any sort of update anyways.

TL;DR: Seems to work fine in 52.2.0 despite what strict_min_version says, but manually editing that and installing it as an unverified extension likely has several downsides not related to the extension itself.
